gradle.properties

```diff
@@ -14,7 +14,7 @@ org.gradle.parallel=true
 
 GROUP=com.amazonaws
 VERSION_NAME=2.7.8
```

Comment:
Update version number to 2.7.9 and also update it in the Version.kt file

Reply:
Updated the version.